vers-one/EpubReader
# Fix for EPUB 2 NCX navigation list parsing

# Fix for EPUB 2 NCX navigation list parsing

This is:

Related issue: #55

## Description

This pull request fixes the EPUB 2 NCX navigation list parsing issues described in the issue linked above. It also adds unit tests for the EPUB 2 NCX reader.

## Testing steps

1. Open a EPUB 2 book which has at least one navigation list in the NCX file.
2. Check the `book.Schema.Schema.Epub2Ncx.NavLists[0].NavigationTargets` property and make sure it contains at least one element.
